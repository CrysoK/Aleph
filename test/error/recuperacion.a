// Programa inv√°lido (debe dar error)
if 1 {
  println('Bien A 1/2')
  println('Mal? A 1/2') println('Mal? A 2/2')
  println('Bien A 2/2')
}
if 0 {
  println('Mal? B 1/1')-
} else {
  println('Bien B 1/1')
}
println('Mal? C 1/2') println('Mal? C 2/2')
println('Bien C 1/2'); println('Bien C 2/2')

println('Mal? D 1/3') println('Mal? D 2/3'); println('Mal? D 3/3')
println('Bien D 1/3'); println('Bien D 2/3'); println('Bien D 3/3')
