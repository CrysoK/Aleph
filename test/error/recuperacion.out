ERROR [Sintaxis] (test/error/recuperacion.a:3:23): Inesperado: identificador. Se esperaba: fin de línea, ';', '}'.
Bien A 1/2
Mal? A 1/2
Bien A 2/2
ERROR [Sintaxis] (test/error/recuperacion.a:7:23): Inesperado: fin de línea.
Bien B 1/1
Mal? C 1/2
ERROR [Sintaxis] (test/error/recuperacion.a:11:23): Inesperado: identificador. Se esperaba: fin de línea, ';'.
Bien C 1/2
Bien C 2/2
Mal? D 1/3
ERROR [Sintaxis] (test/error/recuperacion.a:14:23): Inesperado: identificador. Se esperaba: fin de línea, ';'.
Mal? D 3/3
Bien D 1/3
Bien D 2/3
Bien D 3/3
AVISO: 4 errores encontrados
